src/main/kotlin/com/fatimacarneiro/estudokotlin/phonenumber/Service.kt

```diff
@@ -0,0 +1,57 @@
+package com.fatimacarneiro.estudokotlin.phonenumber
+
+import com.fatimacarneiro.estudokotlin.DDDException
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class Service(
+        val dao: Dao,
+        val conversor: Conversor
+) {
+
+    @Transactional
+    fun save(form: PhoneNumberForm) {
+        val entity: PhoneNumber = conversor.toConvertFormInEntity(form)
```

Comment:
Perhaps you can change conversor for extensions function

Reply:
fun PhoneNumberForm.toEntity(): PhoneNumber {
        val entity = PhoneNumber()
        entity.ddd = this.ddd
        entity.number = this.number

        return entity
}

form.toEntity()